package.json

```diff
@@ -28,12 +28,31 @@
     "build:win:norelease": "npm run build && electron-builder --win --config --publish never",
     "build:win:prerelease": "npm run build && electron-builder --win --config --publish onTagOrDraft --prerelease",
     "build:win:release": "npm run build && cross-env GH_TOKEN=%SLTT_APP_PAT% electron-builder --win --config --publish always",
+    "build:mac:norelease": "electron-vite build && electron-builder --mac --config --publish never",
     "build:mac": "electron-vite build && electron-builder --mac --config",
+    "build:mac:release": "electron-vite build && electron-builder --mac --config --publish always",
     "build:linux": "electron-vite build && electron-builder --linux --config"
   },
   "build": {
     "appId": "net.sltt-bible.app",
     "productName": "sltt-app",
+    "forceCodeSigning": true,
+    "afterSign": "scripts/afterSign.js",
+    "mac": {
+      "category": "public.app-category.business",
+      "entitlements": "build/entitlements.mac.plist",
+      "entitlementsInherit": "build/entitlements.mac.plist",
+      "hardenedRuntime": true,
+      "gatekeeperAssess": false,
+      "target": [
+        {
+          "target": "dmg",
+          "arch": [
+            "x64"
+          ]
+        }
+      ]
+    },
```

Comment:
was all of this needed to be added here, or could `electron-builder.yml` do most of the heavy lifting?
I've found that sometimes I do NEED to change the `package.json`, but I wanted you to be aware that I think `electron-builder.yml` is supposed to be the main build configuration file.

Reply:
Moved everything in "mac" key except "target" from package.json to electron-builder.yml.

When I tried moving "target" info to .yml file build process appeared to ignore the information (which is odd because I found example claiming it ought to work ... altho when I looked in the documentation I could not find anything to lead me to believe it would work).

Reply:
Yeah, I've had to keep some things in json because of this as well.